internal/core/items.rs

```diff
@@ -952,10 +953,7 @@ impl Item for FocusScope {
         window: &WindowRc,
         self_rc: &ItemRc,
     ) -> InputEventResult {
-        /*if !self.enabled() {
-            return InputEventResult::EventIgnored;
-        }*/
-        if matches!(event, MouseEvent::MousePressed { .. }) && !self.has_focus() {
+        if self.enabled() && matches!(event, MouseEvent::MousePressed { .. }) && !self.has_focus() {
```

Comment:
Interresting,  so we do set the focus on click. I actually thought we didn't. (So this is a compelling reason for a more complex focus policy)